src/v/cluster/tests/partition_allocator_fixture.h

```diff
@@ -64,8 +64,12 @@ struct partition_allocator_fixture {
             .available_memory_gb = 5 * core_count,
             .available_disk_gb = 10 * core_count});
 
-        members.local().apply(
+        auto ec = members.local().apply(
           model::offset(0), cluster::add_node_cmd(0, broker));
+        if (!ec) {
```

Comment:
isn't `if (ec)` the way to check for error in `std::error_code`, that is, if it contains a non-zero value?

Reply:
Ugh yes shameful